src/interface/src/app/treatments/treatment-overview/treatment-overview.component.ts

```diff
@@ -23,10 +27,38 @@ import { BehaviorSubject } from 'rxjs';
   styleUrl: './treatment-overview.component.scss',
 })
 export class TreatmentOverviewComponent {
+  constructor(private treatmentsState: TreatmentsState) {}
+
+  treatmentPlan$ = this.treatmentsState.treatmentPlan$;
   savingStatus$ = new BehaviorSubject<boolean>(false);
   errorSavingName: string | null = null;
+  summary$ = this.treatmentsState.summary$;
+  projectAreas$ = this.summary$?.pipe(
+    map((summary: TreatmentSummary | null) => summary?.project_areas)
+  );
 
-  constructor(private treatmentsState: TreatmentsState) {}
+  totalTreatedStands() {
```

Comment:
can you make this properties, so it doesn't get returned each time this is called again?
`totalTreatedStands$ = return this.projectAreas$?.pipe(.....`
same with totalStands